db_information/database.sql

```diff
@@ -0,0 +1,54 @@
+CREATE DATABASE vivero
+    WITH
+    OWNER = postgres
+    ENCODING = 'UTF8'
+    LC_COLLATE = 'Spanish_Spain.1252'
+    LC_CTYPE = 'Spanish_Spain.1252'
+    TABLESPACE = pg_default
+    CONNECTION LIMIT = -1
+    IS_TEMPLATE = False;
+	
+CREATE TABLE producto (
+	id_producto SERIAL PRIMARY KEY,
+	nombre VARCHAR(50) NOT NULL,
+	descripcion  VARCHAR(50) NOT NULL,
+	precio int,
+	cantidad int
```

Comment:
Considera colocar NOT NULL a estos atributos puesto que pueden ser vitales para la base de datos
```suggestion
	precio int NOT NULL,
	cantidad int NOT NULL
```